### DISCUSSION TITLE
Some models doesn't show in AR mode

### DISCUSSION THREAD

**Discussion by ToniGallardoPixel** [google/model-viewer#1978] [Q&A]
Hi, I'm loading a scene in the model-viewer. 
When I see it in the browser all it's ok, but when click on the AR button, it's open the camera and show the mobile icon , but it's close automatically without any message.
It's any limitations about the triangles or the weight of the models?

This is the code, it's similar than the examples:

`<model-viewer 
            id="room"
            class="viewer"
            src="./modelosAR/room.glb" 
            camera-controls shadow-intensity="1"
            ar ar-modes="scene-viewer webxr quick-look" 
            ios-src="./modelosAR/room.usdz">
        </model-viewer>`

And the model weight is 10MB

Thanks in advance.

**Comment by elalish** [comment]
This is a SceneViewer issue, so it's hard to say. Does it work with WebXR? Since WebXR mode continues to use our code, it is faster and generally more reliable. Is there a reason you prefer SceneViewer to WebXR?

**Reply by ToniGallardoPixel** [reply]
Thanks @elalish. Not especially, I took the code from the examples page and didn't pay attention at this point. Sorry.
I will try again.
Thanks
